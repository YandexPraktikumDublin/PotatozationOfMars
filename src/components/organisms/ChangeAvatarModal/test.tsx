import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ChangeAvatarModal } from '.'

describe('<ChangeAvatarModal />', () => {
  const onSuccessAvatarUpdate = jest.fn()

  it('should renders correct <ChangeAvatarModal />', () => {
    const toggleModal = jest.fn()

    const wrapper = shallow(
      <ChangeAvatarModal
        toggleModal={toggleModal}
        onSuccessAvatarUpdate={onSuccessAvatarUpdate}
      />
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
