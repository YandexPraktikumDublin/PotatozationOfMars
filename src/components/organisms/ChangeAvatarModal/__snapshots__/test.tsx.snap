// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChangeAvatarModal /> should renders correct <ChangeAvatarModal /> 1`] = `
<Modal
  toggleModal={[MockFunction]}
>
  <Title>
    Upload avatar
  </Title>
  <ChangeAvatarForm
    successCallback={[MockFunction]}
  />
</Modal>
`;
